spec/features/user_views_navbar_spec.rb

```diff
@@ -16,17 +16,13 @@
 
   describe "as a logged in user" do
     it "should see user name in nav bar" do
-      visit signup_path
-      fill_in "Name", with: "mark"
-      fill_in "Email", with: "m@m.com"
-      fill_in "Password", with: "password"
-      fill_in "Password confirmation", with: "password"
-      click_on "Create account"
+      user = create(:user)
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
```

Comment:
@jwpincus this is how you can stub out a logged in user.

Create a user via a factory, then say say 
> any time we call `ApplicationController.current_user`, give us `user`